lua/modules/menu.lua

```diff
@@ -610,6 +610,7 @@ function actionColumnUpdateContent()
 	else
 		likeIcon.Text = "🤍"
 	end
+	likeLabel.parentDidResize()
```

Comment:
It's better to call it with `:` as `parentDidReize` receives `self`. 
Even if not used here, better shows what should be done in other situations.